frontend/app/services/personal-information-service.server.ts

```diff
@@ -185,6 +188,22 @@ function createPersonalInformationService() {
       },
     };
   }
+  async function getPersonalInformationIntoSession(session: Session, request: Request, redirectURL: string, sin?: string) {
```

Comment:
This function doesn't belong in the service layer; it's the responsibility of the routes to manage this logic. The role of the service layer is to facilitate the transfer of data between the frontend and the backend.